packages/settings/setters/toggles/toggle.js

```diff
@@ -1,29 +1,40 @@
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-
-import { settingsUpdated } from '@common/store/actions';
+import { toast } from 'react-toastify';
+import useSettings from '@settings/managers/useSettings';
+import Piwik from '@common/utilities/piwik';
+import CheckBox from '@basics/inputs/checkBox';
 import Tooltip from '@basics/tooltip/tooltip';
 import Icon from '@basics/icon';
 import styles from '@shared/navigationBars/topBar/topBar.css';
-
 /**
  * Toggles boolean values on store.settings
  *
  * @param {String} setting The key to update in store.settings
  * @param {Array} icons [activeIconName, normalIconName]
  * @param {Array} tips [activeTip, normalTip]
+ * @param {boolean} isCheckbox show checkbox or tooltip
  */
 const Toggle = ({
-  setting, icons, tips,
+  setting, icons, tips, isCheckbox, t,
```

Comment:
we always need the default props
```
Toggle.defaultProps = {
  isCheckbox: false // or true depend on the logic on this case because the backward compatibility should be false
}
  
  export default Toggle
  ```